src/content/docs/merchants/get-started/enrichment.mdx

```diff
@@ -17,6 +17,122 @@ import Link from '@components/Link.astro';
 
 This topic introduces you to the concept of enriching product pages in your storefront. You will learn how to add content blocks above or below a commerce block (or any other block).
 
+<Aside type="caution">
+**Critical Setup Required**: Before using enrichment functionality, you must configure your `query.yaml` file to automatically generate the `enrichment.json` file. This is the most common cause of enrichment failures.
+
+**Common mistake**: Users often manually create the `enrichment.json` file when they see it's missing, but this approach will not work reliably. The file must be generated automatically by the `query.yaml` configuration.
+</Aside>
+
+## Prerequisites
+
+Before you can use enrichment functionality, you must complete the following setup steps:
+
+<Tasks>
+
+<Task>
+### Configure query.yaml to generate enrichment.json
+
+The `enrichment.json` file is automatically generated by the system when you configure your `query.yaml` file properly. You do not need to manually create this file.
+
+Configure your `helix-query.yaml` or the `query.yaml` pushed to your Helix site config to include the enrichment index configuration.
+
+<Code code={`version: 1
+indices:
+  enrichment:
+    target: /enrichment/enrichment.json
+    include:
+      - '**/enrichment/**'
+    properties:
+      title:
+        select: head > meta[property="og:title"]
+        value: |
+          attribute(el, 'content')
+      products:
+        select: head > meta[name="enrichment-products"]
+        values: |
+          match(attribute(el, 'content'), '([^,]+)')
+      categories:
+        select: head > meta[name="enrichment-categories"]
+        values: |
+          match(attribute(el, 'content'), '([^,]+)')
+      positions:
+        select: head > meta[name="enrichment-positions"]
+        values: |
+          match(attribute(el, 'content'), '([^,]+)')`} lang="yaml" title="query.yaml enrichment index configuration" />
+
+</Task>
+
+<Task>
+### Choose the correct location for your query.yaml
+
+You have two options for where to place your `query.yaml` configuration:
+
+**Option 1: Site-specific configuration (Recommended for Helix 5)**
+- Configure at: `https://admin.hlx.page/config/[ORG]/sites/[SITE]/content/query.yaml`
+- This is the recommended approach for new Helix 5 projects
+- Each site has its own independent configuration to prevent conflicts between different sites
+
+**Option 2: GitHub-based configuration**
+- Place as `helix-query.yaml` in your GitHub repository
+- This configuration will be shared among all sites using the same code reference
+- Suitable for multi-site deployments with shared configuration
+- **Note**: All sites using the same code reference will share this configuration
+
+**Important**: Do not use `query-index.json` files for enrichment configuration. The `query-index.json` approach is for Google/SharePoint only and does not automatically generate the required `query.yaml` configuration for Helix sites.
```

Comment:
I don't know what this means - can you provide a reference for this line?

Reply:
I read that part of the thread a few times and it was ambiguous. Maybe we should remove it. It doesn't really add anything that's not mentioned in the rest of the topic.

Reply:
Removed.